laravel/app/Http/Controllers/Administration/PhpInfoController.php

```diff
@@ -12,6 +12,34 @@ class PhpInfoController extends Controller
      */
     public function php_info(): View
     {
-        return view('administration.phpinfo');
+        //embedded php info
+        ob_start();
+        phpinfo();
+        $phpInfo = ob_get_contents();
+        ob_end_clean();
+        $phpInfo = preg_replace('%^.*<body>(.*)</body>.*$%ms', '$1',$phpInfo);
+        $style = /** @lang text */
```

Comment:
the {! phpinfo() !} have there own CSS Stylesheet an this will destroy this template site.
Notes:
https://stackoverflow.com/questions/52865732/how-to-embed-phpinfo-within-a-page-without-affecting-that-pages-css-styles

https://www.php.net/manual/en/function.phpinfo.php

Reply:
Yes, I understood this.

But the definition of `$style` should not be in this PHP controller. It should be rather instead directly in the respective view / blade.

Reply:
Now i understand :-)